piplicenses.py

```diff
@@ -58,12 +58,9 @@
 
 __pkgname__ = "pip-licenses"
 __version__ = "4.5.0"
-__author__ = "raimon"
-__license__ = "MIT"
 __summary__ = (
```

Comment:
Do we really need the summary and version defined twice?

Reply:
These strings are used in CLI interface for help messages and `--version` output, we can try to find ways around it, but only if we really want to do that

Reply:
And package name is used to exclude it from licences analysis

Reply:
I have tried and found that if I declare `[tool.setuptools.dynamic]` in the `pyproject.toml` file, I can package it by reference to `__version__`.

Seems better than declaring it twice.

```diff
 [project]
 name = "pip-licenses"
-version = "4.4.0"
+dynamic = ["version"]
 description = "Dump the software license list of Python packages installed with pip."
 readme = "README.md"
 requires-python = ">=3.8"
@@ -60,6 +60,9 @@ pip-licenses = "piplicenses:main"
 [tool.setuptools]
 py-modules = ["piplicenses"]

+[tool.setuptools.dynamic]
+version = {attr = "piplicenses.__version__"}
+
 [tool.pytest.ini_options]
 addopts = "--pycodestyle -v --cov --cov-report term-missing"
```

Reply:
@raimon49 cool, I'll fix it

Reply:
I've managed to make it read version and combined readme from files, but for "short description" it need the whole text file, you can't read it as python file attribute as in version. As this string will not change much I suggest having it duplicated, not ideal, but...